libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Base.hs

```diff
@@ -585,15 +580,15 @@ nameGovAction UpdateCommittee {} = "UpdateCommittee"
 nameGovAction NewConstitution {} = "NewConstitution"
 nameGovAction InfoAction {} = "InfoAction"
 
-instance IsConwayUniv fn => ExecSpecRule fn "EPOCH" Conway where
+instance fn ~ ConwayFn => ExecSpecRule fn "EPOCH" Conway where
```

Comment:
This looks suspicious, like some where there is a function that requires
'fn' to be ConwayFn, when it ought to instead have have (IsConwayUniv fn) as a constraint.

Reply:
Added a comment